### PR TITLE
Feature/fix image repo setting on install mds

### PR DESCRIPTION
**Pull request by lucasnoah**
## PR Checklist

 - [ ] simple searchable title - `[mds-db] Add PG env var`, `[config] Fix eslint config`
 - [ ] briefly describe the changes in this PR
 - [ ] mark as draft if should not be merged
 - [ ] write tests for all new functionality

## Impacts
- [ ] Provider
- [ ] Agency
- [ ] Audit
- [ ] Policy
- [ ] Compliance
- [ ] Daily
- [ ] Native
- [ ] Policy Author

### CODE REVIEW

**Comment by lucasnoah** [comment]
@jwtodd I originally went with your initial inline version, but I swapped it up because the gcr repo images lack appropriate version tags. The only tag I saw across everything was latest. We could repush to the gcr repo with appropriate tags if we want to go this route.

**Comment by jwtodd** [comment]
ahhh ... the image tag occurs in build() and if the image-reposistory config exists an ensuing push will take place. re versioned images, there is another grc (mds-automation) that includes versions (and i can update with the latest). does this help? worse case, let's go with your patch as is 'cuz ... it works for the operational mode you require.

File: bin/mdsctl [hunk 1]
```diff
@@ -503,12 +503,13 @@ imageVersion() {
 
 installMds() {
   check ${FUNCNAME[0]}
-
   installFallbackCertificate
+
+  [ -z "${configs[image-repository]}" ] && configure $(imageVersion) || configure sets-mds+=registry=${configs[image-repository]} && configure $(imageVersion)
```
[the remark this answers is not in the document]
Comment (janedotx) [reply bin/mdsctl]: Since #104 has been merged, can this be closed now?

[the remark this answers is not in the document]
Comment (jwtodd) [reply bin/mdsctl]: @janedotx yeah let's close this one

**Pull request closed without merging**